### DISCUSSION TITLE
Reduction of accelerations depending on the height of the product

### DISCUSSION THREAD

**Discussion by Kawasalto** [OrcaSlicer/OrcaSlicer#4969] [Ideas]
Hi. Is there a setting to decrease acceleration depending on the height of the product? After all, the higher the product and the thinner it is, the more it begins to chatter from inertia at high accelerations.

**Comment by koriwi** [comment]
I implemented this myself in Klipper with the tuning tower command. I have overwrote the command that sets the acceleration so it multiplies the acceleration with a modifier.  Then I have a tuning tower command in my start print macro which sets this modifier value.  Using it for more than a year alreAdy and it works perfectly

**Reply by vgdh** [reply]
good workaround!

**Reply by tektalon** [reply]
Would you mind sharing your command ?

**Comment by MauroOrlic** [comment]
I'd love to have this as a setting for my bedslinger. While printing tall skinny objects the bottom will be perfect while towards the top there is low frequency ringing due to the object itself flexing, like a tall building in the wind. Call it something like "Exponential acceleration decay" that is calculated A(h)=A0*(1−r)^h where h is current layer height in mm, A(h) acceleration at that height, A0 initial acceleration, and r the acceleration reduction in %, the parameter.

For example, with initial acceleration of 8000mm/s2, Exponential acceleration decay set to 0.5%, at 100mm and 200mm height you would have:
 A(100mm)=8000mm/s2*(1-0.5%)^100mm = 4846mm/s2
 A(200mm)=8000mm/s2*(1-0.5%)^200mm = 2936mm/s2

For EAD 0.1% and 1%:
A(100mm)=8000mm/s2*(1-0.1%)^100mm = 7238mm/s2
A(200mm)=8000mm/s2*(1-0.1%)^100mm = 6549mm/s2

 A(100mm)=8000mm/s2*(1-1%)^100mm = 2928mm/s2
 A(200mm)=8000mm/s2*(1-1%)^100mm = 1072mm/s2

Since ideal EAD depends on rigidity of the material (and geometry of the part, but lets ignore that for now), this should probably be under Filament settings. Lower values suitable for stiff materials like PLA, higher values for flexible materials like TPU.
